README.md

```diff
@@ -106,52 +147,43 @@ for example in examples:
     result = circuit.encrypt_run_decrypt(*example)
     print(f"Evaluation of {' + '.join(map(str, example))} homomorphically = {result}")
 ```
+*This example is explained in more detail [in this part of the documentation](https://docs.zama.ai/concrete/getting-started/quick_start).*
 
-## Documentation
-
-Full, comprehensive documentation is available at [https://docs.zama.ai/concrete](https://docs.zama.ai/concrete).
-
-## Target users
-
-Concrete is a generic library that supports a variety of use cases. Because of this flexibility,
-it doesn't provide primitives for specific use cases.
-
-If you have a specific use case, or a specific field of computation, you may want to build abstractions on top of Concrete.
+<p align="right">
+  <a href="#table-of-contents" > ↑ Back to top </a> 
```

Comment:
what's the rule for Back to Top? is it one from time to time (by opposition to one per section)

Reply:
ok thx